sky/shell/platform/mojo/platform_view_mojo.cc

```diff
@@ -147,5 +156,23 @@ void PlatformViewMojo::OnEvent(mojo::EventPtr event,
   callback.Run();
 }
 
+void PlatformViewMojo::Create(
+    mojo::ApplicationConnection* connection,
+    mojo::InterfaceRequest<::keyboard::KeyboardService> request) {
+
+  mojo::ServiceProviderPtr sp;
+  connector_->ConnectToApplication("mojo:keyboard", mojo::GetProxy(&sp), nullptr);
+
+#ifdef OS_LINUX
```

Comment:
Eventually both implementations will need to provide context.  I have another pass I'd like to do on that code that will make the ifdef go away entirely.